python/appledoublekaitai/testdata/result/_Keynote2013-Sample09.key.apple_double_kaitai_plugin.0.text

```diff
@@ -0,0 +1,65 @@
+{
+  "magic": {
+    "name": "APPLE_DOUBLE",
+    "value": 333319
+  },
+  "version": 131072,
+  "reserved": [
+    77,
+    97,
+    99,
+    32,
+    79,
+    83,
+    32,
+    88,
+    32,
+    32,
+    32,
+    32,
+    32,
+    32,
+    32,
+    32
+  ],
+  "numEntries": 2,
+  "entries": [
+    {
+      "type": {
+        "name": "FINDER_INFO",
+        "value": 9
+      },
+      "ofsBody": 50,
+      "lenBody": 127,
+      "body": {
+        "fileType": [
```

Comment:
(a bit out of scope, but nice if we can resolve this)
fileType is an enum defined in the Kaitai struct, here we print the raw value.
in line with @Schramp s comment: can we get the textual representation of the enum?